docs/preview/02-Features/02-message-handling/04-event-hubs-azure-functions.md

```diff
@@ -0,0 +1,242 @@
+﻿---
+title: "Azure EventHubs message handling for Azure Functions"
+layout: default
+---
+
+This article describes how you can use Arcus' message handler concept with Azure Functions; allowing you to more easily port your business logic from/to Azure Functions.
+
+# Azure EventHubs message handling for Azure Functions
+While our default message pump system provides a way to receive, route, and handle incoming EventHubs messages which are, unfortunately, not supported in Azure Functions.
+Today, Azure Functions acts as a message receiver meaning that the function is triggered when a message is available but does not handle message routing and handling. However, in this case, it acts as the message pump.
+
+Following terms are used:
+- **Message handler**: implementation that processes the received message from an Azure EventHubs subscription. Message handlers are created by implementing the `IAzureEventHubsMessageHandler<TMessage>`. This message handler will be called upon when a message is available on the Azure EventHubs subscription.
+- **Message router**: implementation that delegates the received Azure EventHubs message to the correct message handler.
+
+That's why we extracted our message routing functionality so you can call it directly from your Azure Function.
+
+![Azure Functions message handling](/media/az-func-eventhubs-message-handling.png)
+
+We will walk you through the process of using message handlers with Azure Functions:
+
+## Installation
+To use the following described features, install the following package:
+```shell
+PM > Install-Package -Name Arcus.Messaging.AzureFunctions.EventHubs
+```
+
+## Receive Azure EventHubs message in an Azure Function
+Here's an example of how an Azure Function receives an Azure EventHubs message from a topic:
+
+```csharp
+public class SensorReadingFunction
+{
+    [Function("sensor-reading")]
+    public async Task Run(
+        [EventHubTrigger("sensors", Connection = "EventHubsConnectionString")] string[] messages,
+        Dictionary<string, JsonElement>[] propertiesArray,
+        FunctionContext context)
+    {
+        // Processing events...
+    }
+}
+```
+
+## Declaring our Azure EventHubs message handlers
+Registering message handlers to process the EventHubs message happens just the same as using a message pump.
```

Comment:
With 'message pump', are you referring to how we are handling / routing ServiceBus messages ?
Maybe we should simplify this sentence, and not refer to the 'message pump'.
Something like

> Registering message handlers to process the EventHubs events is fairly easy to do.  Here is an example on how you can register message-handlers in your Function:

(Actually, I believe that the MS docs always talk about events instead of messages, when covering eventhubs).

Reply:
Changed the sentence with your proposal and updated 'messages' to 'events'.